trin-core/src/jsonrpc/endpoints.rs

```diff
@@ -43,14 +43,19 @@ pub enum TrustedProviderEndpoint {
     BlockNumber,
 }
 
-/// Ethereum JSON-RPC endpoints supported by portal network requests
+/// Ethereum JSON-RPC endpoints supported by portal network requests (eg. `eth_*`)
 #[derive(Debug, PartialEq, Eq, Clone)]
-pub enum PortalEndpoint {
-    ClientVersion, // Doesn't actually rely on portal network data, but it makes sense to live here
+pub enum EthEndpoint {
     GetBlockByHash,
     GetBlockByNumber,
 }
 
+/// General Trin endpoints that don't rely on fetching portal network data
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub enum PortalEndpoint {
+    ClientVersion,
+}
```

Comment:
This enum will also be useful for housing the generic paginate local content keys endpoint in #474